src/pybind/mgr/feedback/module.py

```diff
@@ -0,0 +1,110 @@
+
+"""
+Feedback module
+
+See doc/mgr/feedback.rst for more info.
+"""
+
+from mgr_module import CLIReadCommand, HandleCommandResult, MgrModule
+import errno
+from .services import CephTrackerClient
+from .model import Feedback
+
+
+class FeedbackModule(MgrModule):
+
+    # there are CLI commands we implement
+    @CLIReadCommand('set issue_key')
+    def set_api_key(self, key: str) -> HandleCommandResult:
+        """
+        Set Ceph Issue Tracker API key
+        """
+        try:
+            self.set_store('issue_tracker', key)
+        except Exception as error:
+            return HandleCommandResult(stderr=f'Exception in setting API key : {error}')
+        return HandleCommandResult(stdout="Successfully updated key")
+
+    @CLIReadCommand('delete issue_key')
+    def delete_api_key(self) -> HandleCommandResult:
+        """
+        Delete Ceph Issue Tracker API key
+        """
+        try:
+            self.set_store('issue_tracker', None)
+        except Exception as error:
+            return HandleCommandResult(stderr=f'Exception in deleting API key : {error}')
+        return HandleCommandResult(stdout="Successfully deleted key")
+
+    def is_api_key_set(self):
+        try:
+            key = self.get_store('issue_tracker')
+            if key is None:
+                return False
+            return key != ''
+        except:
+            return False
+
+    @CLIReadCommand('get issue_key')
+    def get_api_key(self) -> HandleCommandResult:
+        """
+        Get Ceph Issue Tracker API key
+        """
+        try:
+            key = self.get_store('issue_tracker')
+            if key is None:
+                return HandleCommandResult(stderr='Issue tracker key is not set. Set key with `ceph set issue_key <your_key>`')
+        except Exception as error:
+            return HandleCommandResult(stderr=f'Error in retreiving issue tracker API key: {error}')
+        return HandleCommandResult(stdout=f'Your key: {key}')
+
+    @CLIReadCommand('get issue')
```

Comment:
Is this listing all issues? In that very case it should be:
```suggestion
    @CLIReadCommand('feedback issue list')
```